src/pages/Filing/ViewInstitutionProfile/FinancialInstitutionDetails.tsx

```diff
@@ -25,8 +24,11 @@ export function FinancialInstitutionDetails({
       <Heading type='2'>{heading}</Heading>
       <Paragraph>
         To make a change to the email domains for your financial institution,{' '}
-        <Link href={sblHelpLink}>contact our support staff</Link>. To make a
-        change to any other data in this section, visit <Links.GLIEF />.
+        <Link href='mailto:SBLHelp@cfpb.gov?subject=[BETA] View your financial institution profile: updating my financial institution information'>
```

Comment:
This component also gets rendered in the `Update institution` form, so a more generic `Subject` may be more fitting. 
```suggestion
        <Link href='mailto:SBLHelp@cfpb.gov?subject=[BETA] Financial institution profile: updating my financial institution information'>
```

Reply:
Great catch @meissadia! 🎉

[Fixed](https://github.com/cfpb/sbl-frontend/pull/330/commits/97df326c0bd2a38fdde0c3781b2ed5dbe6de9e5a)!